### DISCUSSION TITLE
Would it be possible to support 0 and 1 as boolean values

### DISCUSSION THREAD

**Discussion by scalpel4k** [stephenberry/glaze#978] [Ideas]
I know it's not officially supported to use 0 and 1 as boolean values. However, reality looks a bit different and I know a few situations especially in the embedded world where this is common practice.

It would be really cool to have a compile time option to convert between numeric values and booleans. It would be way easier than writing custom converters all the time....

thx Michi

**Comment by stephenberry** [comment]
I think this is a reasonable and simple idea for a compile time option and wrapper. I'll convert this to an issue to get it supported.

**Comment by stephenberry** [comment]
Support has been added in #1022 . You can use the compile time option `bools_as_numbers = true` or the wrapper by the same name.
